lib/Resolver.js

```diff
@@ -78,6 +78,19 @@ Resolver.prototype.resolve = function resolve(context, path, request, callback)
 	}, null));
 };
 
+Resolver.prototype.doResolveWithErrorLogging = function doResolveWithErrorLogging(type, request, message, callback, options) {
+	var optionsWithoutLog = {
+		stack: options.stack,
+		missing: options.missing
+	};
+
+	this.doResolve(type, request, message, createInnerCallback((err, result) => {
```

Comment:
That makes sense.

This is probably a micro optimization that likely has little to no impact, but I was just thinking that we wouldn't need the `callbackWrapper.stack` and `callbackWrapper.missing` with this code path anyway, since we recall `this.doResolve` w/ `createInnerCallback` on the error case.